dev/benchmarks/platform_channels_benchmarks/.gitignore

```diff
@@ -0,0 +1,46 @@
+# Miscellaneous
```

Comment:
Does this need its own gitignore? We have one on the toplevel with what appears to be similar content.

Reply:
removed